cpp/include/cuml/decomposition/tsvd_mg.hpp

```diff
@@ -46,7 +46,7 @@ void fit(raft::handle_t& handle,
 
 void fit(raft::handle_t& handle,
          MLCommon::Matrix::RankSizePair** rank_sizes,
-         size_t n_parts,
+         std::size_t n_parts,
```

Comment:
All of the `n_parts` should stay 32-bits (ideally unsigned)

Reply:
I just wanted to add the std namespace since it was already 64-bits, but it's true that this can become unsigned 32. I made the changes.